.travis.yml

```diff
@@ -3,20 +3,26 @@ sudo: required
 before_script:
   - npm install -g bower polymer-cli@next
   - polymer install --variants
+  - sudo mv /usr/bin/google-chrome /usr/bin/google-chrome-old
+  - sudo mv /usr/bin/google-chrome-beta /usr/bin/google-chrome
 env:
   global:
-    - secure: JibMotHxRDu0L2Fe4raETzvCmvO1CQ0gG7O0XhfGArhuwTC+IW7MLlO/wYpVUkfCLoSKDvIiLSYKpPtUPJ+kZy04VENNc/axKcPdr4iu54se+qe61mRmAunMBVrS21SKlxnqflO1iPFyJ71IyD0/hGBY76Chj4AMUDWhrHJ0WjE=
-    - secure: qEULPMDsjwPoNkedrd9azBPPTxckC5hWGKAD8hceTOhoFTxNKKkQmB/Hpx+DomAWXq9yFflB9nRqSTnw9la+Py2bHPWohD9tz+YVsZPvhJmTugrOL+i8B7IzCtncCm7WfWDyjxQFYaHg+UIiEcg9Ci29sNAsWnKakl82e1FJ8oU=
+    - secure: >-
+        JibMotHxRDu0L2Fe4raETzvCmvO1CQ0gG7O0XhfGArhuwTC+IW7MLlO/wYpVUkfCLoSKDvIiLSYKpPtUPJ+kZy04VENNc/axKcPdr4iu54se+qe61mRmAunMBVrS21SKlxnqflO1iPFyJ71IyD0/hGBY76Chj4AMUDWhrHJ0WjE=
+    - secure: >-
+        qEULPMDsjwPoNkedrd9azBPPTxckC5hWGKAD8hceTOhoFTxNKKkQmB/Hpx+DomAWXq9yFflB9nRqSTnw9la+Py2bHPWohD9tz+YVsZPvhJmTugrOL+i8B7IzCtncCm7WfWDyjxQFYaHg+UIiEcg9Ci29sNAsWnKakl82e1FJ8oU=
```

Comment:
This is how the Travis config is formatted on all the 2.0-preview branches, so I figured I keep it consistent:

https://github.com/PolymerElements/iron-doc-viewer/blob/2.0-preview/.travis.yml
https://github.com/PolymerElements/paper-button/blob/2.0-preview/.travis.yml